globus_automate_client/cli/flows.py

```diff
@@ -345,7 +419,12 @@ def flow_list(
     flows = fc.list_flows(
         roles=[r.value for r in roles], marker=marker, per_page=per_page
     )
-    format_and_echo(flows, verbose=verbose)
+
+    format_and_echo(
+        flows,
+        json.dumps if output_format == FlowDisplayFormat.json else yaml.dump,
```

Comment:
I'm fine with this method of determining which dump function to use or the if/else combination used above. May want to standardize on one or the other, though.